app/src/main/java/org/wikipedia/descriptions/DescriptionEditTutorialFragment.kt

```diff
@@ -42,9 +43,10 @@ class DescriptionEditTutorialFragment : OnboardingFragment() {
     companion object {
         const val ARG_POSITION = "position"
         val pages = arrayOf(R.layout.inflate_description_edit_tutorial_page_one, R.layout.inflate_description_edit_tutorial_page_two)
+        val AiIncludedPages = arrayOf(R.layout.inflate_description_edit_tutorial_page_one, R.layout.inflate_description_edit_tutorial_page_two, R.layout.inflate_description_edit_tutorial_page_three)
```

Comment:
Would it be possible to make the `pages` includes all layouts so we can handle the logic only in `getItemCount()`?

```
       override fun getItemCount(): Int {
            return if (showAIOnBoarding) pages.size else pages.size - 1
        }
```

Reply:
Good idea!